build_tools/python_deploy/build_linux_packages.sh

```diff
@@ -44,7 +44,7 @@ TM_RELEASE_DOCKER_IMAGE="${TM_RELEASE_DOCKER_IMAGE:-stellaraccident/manylinux201
 # ./build_tools/docker/Dockerfile
 TM_CI_DOCKER_IMAGE="${TM_CI_DOCKER_IMAGE:-powderluv/torch-mlir-ci:latest}"
 # Version of Python to use in Release builds. Ignored in CIs.
-TM_PYTHON_VERSIONS="${TM_PYTHON_VERSIONS:-cp39-cp39 cp310-cp310}"
+TM_PYTHON_VERSIONS="${TM_PYTHON_VERSIONS:-cp37-cp37 cp39-cp39 cp310-cp310}"
```

Comment:
This should be `cp37-cp37m`

Reply:
It also says "Ignored in CI's" -- do we need to update somewhere else for the nightly packages?

Reply:
Done, thanks.  Is there a reference somewhere I can look at to understand this?

Reply:
Yeah we don't use TM_PYTHON_VERSIONS for CIs since our CI run on Ubuntu 22.04 with Python 3.10. These versions come from the naming scheme using my manylinux (and they have it as cp37-cp37m).  The `m` says it is Python 3.7 with the pymalloc extension (https://peps.python.org/pep-3149/#proposal). Later versions have it integrated by default.